src/syrupy/extensions/json/__init__.py

```diff
@@ -0,0 +1,155 @@
+import datetime
+import json
+from gettext import gettext
+from pathlib import Path
+from types import GeneratorType
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Set,
+)
+
+from syrupy.constants import SYMBOL_ELLIPSIS
+from syrupy.data import SnapshotFossil
+from syrupy.extensions.amber.serializer import Repr
+from syrupy.extensions.single_file import SingleFileSnapshotExtension
+
+if TYPE_CHECKING:
+    from syrupy.types import (
+        PropertyFilter,
+        PropertyMatcher,
+        PropertyPath,
+        SerializableData,
+        SerializedData,
+    )
+
+
+class JSONSnapshotExtension(SingleFileSnapshotExtension):
+    _max_depth: int = 99
```

Comment:
There's room to clean this up but can revisit with the larger API changes as part of syrupy v2